contracts/governance/ValidatorSetSig.sol

```diff
@@ -113,7 +117,9 @@ contract ValidatorSetSig is ReentrancyGuard {
         // We don't need to protect against return bomb vectors below here since the caller is expected to have full control over the contract called.
         (bool success,) =
         // solhint-disable-next-line avoid-low-level-calls
-         validatorSetSigMessage.targetContractAddress.call(validatorSetSigMessage.payload);
+        validatorSetSigMessage.targetContractAddress.call{value: validatorSetSigMessage.value}(
```

Comment:
do you know what error gets shown if `this` balance doesn't have enough funds?

Reply:
Internally  when testing it's `EvmError: OutOfFunds` but then  `bool success` is false so the actual revert message is `ValidatorSetSig: call failed`